src/Spice86.Core/Emulator/InterruptHandlers/Input/Keyboard/BiosKeyboardInt9Handler.cs

```diff
@@ -11,9 +12,9 @@
 public class BiosKeyboardInt9Handler : InterruptHandler {
     private readonly Keyboard _keyboard;
 
-    public BiosKeyboardInt9Handler(Machine machine, ILoggerService loggerService) : base(machine, loggerService) {
+    public BiosKeyboardInt9Handler(Machine machine, BiosDataArea biosDataArea, ILoggerService loggerService) : base(machine, loggerService) {
         _keyboard = machine.Keyboard;
-        BiosKeyboardBuffer = new BiosKeyboardBuffer(machine.Memory);
+        BiosKeyboardBuffer = new BiosKeyboardBuffer(machine.Memory, biosDataArea);
```

Comment:
The memory is only passed in in case a game moves the keyboard buffer somewhere else right?